c2corg_ui/views/document.py

```diff
@@ -221,3 +222,13 @@ def _diff(self):
             return self.template_input
 
         raise HTTPNotFound()
+
+    notFound = 'c2corg_ui:templates/404.html'
+
+    @view_config(context=HTTPInternalServerError, renderer=notFound)
+    @view_config(context=HTTPBadRequest, renderer=notFound)
+    @notfound_view_config(renderer=notFound)
+    def notfound(self):
```

Comment:
There are two ways you can declare views with Pyramid, either as [a method of a class](http://docs.pylonsproject.org/projects/pyramid/en/latest/narr/viewconfig.html#view-defaults-class-decorator) or as a [distinct function](http://docs.pylonsproject.org/projects/pyramid/en/latest/narr/viewconfig.html#adding-view-configuration-using-the-view-config-decorator). The error message sounds like you tried to mix the two.